pkg/apis/pipeline/v1/container_types.go

```diff
@@ -541,3 +570,32 @@ func (s *Sidecar) SetContainerFields(c corev1.Container) {
 	s.StdinOnce = c.StdinOnce
 	s.TTY = c.TTY
 }
+
+// GetVarSubstitutionExpressions walks all the places a substitution reference can be used
+func (s *Sidecar) GetVarSubstitutionExpressions() []string {
```

Comment:
please add tests for this exported function 

https://github.com/tektoncd/community/blob/main/standards.md#go-packages